### ISSUE TITLE
flymake count stopped working

### ISSUE THREAD

**Issue by jangid** [joaotavora/eglot#588]
I am using Emacs from `master' branch. I am on macOS Big Sur.

1. Flymake is always showing [0 0] in buffers managed by Eglot. In other buffers, Flymake is showing correct count.
2. Surprisingly, only the count is not updated, inside the buffer window the flymake diagnostics are visible and flymake-goto-next/prev is also working.

Has anyone else observed this behaviour on `emacs-master'?

**Comment by joaotavora** [comment]
I'm also using latest Emacs master.  I did some Flymake updates there recently, specifically to the mode line indicator.

However, I don't see what you're seeing, so likely I need a recipe.  You need to specify what you are doing so I can reproduce and help you, otherwise it's very difficult.  See:

https://github.com/joaotavora/eglot#reporting-bugs

**Comment by jangid** [comment]
Let me isolate the problem and revert.

**Comment by joaotavora** [comment]
Never mind.  I have reproduced the error.

**Comment by joaotavora** [comment]
This is likely in Flymake itself, though I'm not 100% sure yet.

**Comment by joaotavora** [comment]
Fixed upstream in Emacs's flymake.el.  Will be available soon in GNU ELPA Flymake version 1.1.1.

This is the Emacs commit:
```
commit ace749f2e3be5b23cf4cd29c702a0e82d4d871c3 (HEAD -> master, origin/master, origin/HEAD)
Author: João Távora <joaotavora@gmail.com>
Date:   Sat Jan 9 18:49:49 2021 +0000

    Count Flymake diagnostics in modeline by severity, not type

    Originally reported in https://github.com/joaotavora/eglot/issues/588
    by Pankaj Jangid.

    * lisp/progmodes/flymake.el (flymake--mode-line-counter): Count
    diagnostics by severity level, not by type.
    (Version): Bump to 1.1.1
```

**Comment by jangid** [comment]
Verified. Working fine in my environment. Thanks a lot. _/\_
